bin/oli/README.md

```diff
@@ -0,0 +1,89 @@
+# oli
```

Comment:
Those changes should not be included in this PR, maybe you need a rebase?

Reply:
fixed in new commit